ReactiveCocoa/Swift/DynamicProperty.swift

```diff
@@ -5,8 +5,8 @@ import enum Result.NoError
 /// types, including generic types when boxed via `AnyObject`).
 private protocol ObjectiveCRepresentable {
 	associatedtype Value
-	static func extractValue(fromRepresentation representation: AnyObject) -> Value
-	static func represent(value value: Value) -> AnyObject
+	static func extracting(from representation: AnyObject) -> Value
+	static func representing(_ value: Value) -> AnyObject
```

Comment:
Since these are `static func`s, I don't believe they should use the `ing` names.